src/apps/products/routers.py

```diff
@@ -0,0 +1,97 @@
+from fastapi import Depends, status, Response
+from fastapi.routing import APIRouter
+from sqlalchemy.orm import Session
+
+from src.apps.products.schemas import (
+    CategoryInputSchema,
+    CategoryOutputSchema,
+    ProductInputSchema,
+    ProductOutputSchema
+)
+from src.apps.products.services.category_services import (
+    create_category,
+    get_all_categories,
+    get_single_category,
+    update_single_category,
+    delete_single_category
+)
+from src.apps.products.services.product_services import (
+    create_product,
+    get_all_products,
+    get_single_product,
+    update_single_product,
+    delete_single_product
+)
+from src.dependencies.get_db import get_db
+from src.dependencies.user import authenticate_user
+
+
+category_router = APIRouter(prefix="/category", tags=["category"])
```

Comment:
```suggestion
category_router = APIRouter(prefix="/categories", tags=["category"])
```
plural form (REST resource)